src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,20 +1,39 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
 import './MoviesList.scss';
 import { MovieCard } from '../MovieCard';
 import { Movie } from '../../types/Movie';
 
 type Props = {
   movies: Movie[];
+  dublicate: boolean,
 };
 
-export const MoviesList: React.FC<Props> = ({ movies }) => (
-  <div className="movies">
-    {movies.map(movie => (
-      <MovieCard
-        key={movie.imdbId}
-        movie={movie}
-      />
-    ))}
-  </div>
-);
+export const MoviesList: React.FC<Props> = ({ movies, dublicate }) => {
+  const [show, setShow] = useState(true);
+
+  useEffect(() => setShow(true), [movies]);
+
+  return (
+    <div className="movies">
+      {dublicate && show && (
+        <div className="notification">
+          <button
+            className="delete"
+            type="button"
+            onClick={() => setShow(false)}
+          >
+            +
+          </button>
+          its dublicate
+        </div>
+      )}
```

Comment:
I'm not sure you need it, just do not add anything to the list if a movie already exists